### PR TITLE
[FIX] Correct the treeview of the Event Form(#241)

### PR DESCRIPTION
**Pull request by jananjoy**
Correct the tree view of the Event Form

### CODE REVIEW

**Review by rimbendhaou — CHANGES_REQUESTED** [review]
the name of the form is not correct (open: Task and open:Activity)

File: project_event/views/project_project_view.xml [hunk 1]
```diff
@@ -66,16 +70,26 @@
...
-                        }" domain="[('activity_task_type','=', 'activity')]"/>
+							'form_view_ref':'project_event.project_event_activity_form',
+							'tree_view_ref':'project_event.project_event_activity_tree',
+							'default_activity_task_type':'activity',
+							'default_project_id': active_id,
+							}" domain="[('activity_task_type','=', 'activity')]">
+                                <tree>
+                                    <field name="name"/>
+                                    <field name="code"/>
+                                    <field name="parent_id" string="Event"/>
```
Comment (rimbendhaou) [inline project_event/views/project_project_view.xml]: field name="project_id" string="Event"

File: project_event/views/project_project_view.xml [hunk 2]
```diff
@@ -66,16 +70,26 @@
...
+                                    <field name="date_start"/>
+                                    <field name="date_end"/>
+                                    <field name="task_order" invisible="1"/>
+                                    <field name="task_state"/>
```
Comment (rimbendhaou) [inline project_event/views/project_project_view.xml]: string =State

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_event/views/project_project_view.xml (hunk 2 above)
Comment (jananjoy) [reply project_event/views/project_project_view.xml]: @rimbendhaou :+1:  Done

File: project_event/views/project_project_view.xml (hunk 1 above)
Comment (jananjoy) [reply project_event/views/project_project_view.xml]: @rimbendhaou :Done

**Review by rimbendhaou — APPROVED** [review]

**Comment by rimbendhaou** [comment]
@jananjoy please resolve conflits

File: project_event/views/project_project_view.xml [hunk 3]
```diff
@@ -66,16 +70,26 @@
...
+							'default_project_id': active_id,
+							}" domain="[('activity_task_type','=', 'activity')]">
+                                <tree>
+                                    <field name="name"/>
+                                    <field name="code"/>
+                                    <field name="project_id" string="Event"/>
+                                    <field name="date_start"/>
+                                    <field name="date_end"/>
+                                    <field name="task_order" invisible="1"/>
+                                    <field name="task_state" string="State"/>
```
Comment (eilst) [inline project_event/views/project_project_view.xml:85]: task_state => state

Comment (eilst) [reply project_event/views/project_project_view.xml:85]: this changed was already merged last week, you should update your merge

Comment (jananjoy) [reply project_event/views/project_project_view.xml:85]: @eilst : I have resolved the conflicts and change the value accordingly

**Pull request merged**